src/main/java/com/example/accountingapp/enums/Unit.java

```diff
@@ -1,13 +1,21 @@
 package com.example.accountingapp.enums;
 
+import lombok.Getter;
+
+@Getter
 public enum Unit {
-    PIECES("Pieces");
+    LBS("Libre"),
+    GALLON("Gallon"),
+    PCS("Pieces"),
+    KG("Kilogram"),
+    METER("Meter"),
+    INCH("inch"),
+    FEET("Feet");
```

Comment:
I know you are using input from Cihat but what is libre :) ?  pounds ? 
"inch"  should start with Capital or everything should be lower cas 

frankly I would change to standard  acronyms (kg, in , lbs, m, ft, pcs etc ) I think it looks more professional and takes less space
just my opinion - feel free to disagree :)